RELEASE.md

```diff
@@ -10,20 +10,15 @@
 # Upcoming Release 0.18.0
```

Comment:
I just realised that when I open the release notes, I see there's notes about us release 0.17.8 - we should probably remove that section.

Reply:
I suggest we can remove that at release time, otherwise we'll get a bunch of conflicts on `merge-main-to-develop` PRs on this file. 🤔